src/components/Header/styles.module.css

```diff
@@ -1,27 +1,49 @@
 .header {
-  border: 4px solid #253f58;
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  align-items: center;
-  justify-content: space-between;
-  margin: 1rem;
-  padding: 2rem 4rem;
-  text-align: center;
+    display: flex;
```

Comment:
Looks like it converted to 4 spaces. Please use prettier for formatting. If you can't get your editor to work with prettier you can use the prettier cli to format the files before committing them. See `prettier --write` https://prettier.io/docs/en/cli.html

Reply:
works fine, just configured. lots of other files in the repo are off-guidelines I noticed fyi!

Reply:
Not sure if I understand. Can you point to an example?

Reply:
well I ran `prettier --write` from root with the .prettierrc from the repo and it updated a whole bunch of files that I haven't edited: 

<img width="437" alt="Screen Shot 2020-04-16 at 4 26 20 PM" src="https://user-images.githubusercontent.com/1316975/79503363-0a096f80-7fff-11ea-96f3-7e62ae40a4da.png">